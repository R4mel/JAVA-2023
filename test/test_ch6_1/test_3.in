1
15
11
K k1
11
P p0 10 0 true :Jeju:
11
P p0 11 0 true :Jeju:
11
S s1 1 0 false : : Physics 0 1
11
S s1 1 0 false :Busan: Physics 0 2
11
W w1 3 33.3 false :Kangnam-gu Seoul: Samsung Manager
11
W w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager
0
0
