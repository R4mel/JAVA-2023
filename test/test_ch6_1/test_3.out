
******* Main Menu ********
* 0.exit 1.PersonManager *
* 2.ch2 3.ch3 4.ch5      *
**************************
menu item? 1
[0] p0 10 70.0 false :Gwangju Nam-gu Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil, Jongno-gu, Seoul:
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[3] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[4] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
================================================================
menu item? 15
15: OUT of selection range(0 ~ 14) Try again!!

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
================================================================
menu item? 11
input [delimiter(P,S,or W)] [person information] to find by equals():
K k1
K: WRONG delimiter

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
================================================================
menu item? 11
input [delimiter(P,S,or W)] [person information] to find by equals():
P p0 10 0.0 true :Jeju:
[0] p0 10 70.0 false :Gwangju Nam-gu Bongseon-dong 21:

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
================================================================
menu item? 11
input [delimiter(P,S,or W)] [person information] to find by equals():
P p0 11 0.0 true :Jeju:
NOT found by equals()

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
================================================================
menu item? 11
input [delimiter(P,S,or W)] [person information] to find by equals():
S s1 1 0.0 false : : Physics 0.0 1
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
================================================================
menu item? 11
input [delimiter(P,S,or W)] [person information] to find by equals():
S s1 1 0.0 false :Busan: Physics 0.0 2
NOT found by equals()

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
================================================================
menu item? 11
input [delimiter(P,S,or W)] [person information] to find by equals():
W w1 3 33.3 false :Kangnam-gu Seoul: Samsung Manager
NOT found by equals()

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
================================================================
menu item? 11
input [delimiter(P,S,or W)] [person information] to find by equals():
W w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
================================================================
menu item? 0

******* Main Menu ********
* 0.exit 1.PersonManager *
* 2.ch2 3.ch3 4.ch5      *
**************************
menu item? 0

Good bye!!
